Developer-guide/Backport-Guidelines.md

```diff
@@ -1,87 +1,32 @@
-Bugs often get fixed in master before release branches. When a bug is
-fixed in the master branch, it might be desirable or necessary in a
-stable branch. To put the fix in the stable branch we need to backport the
-fix to the stable branch.
+In GlusterFS project, as a policy, any new change, bug fix, etc., are to be
+fixed in 'master' branch before release branches. When a bug is fixed in
+the master branch, it might be desirable or necessary in release branch.
 
-Anyone in the community can suggest a backport. If you are interested to
-suggest a backport, please check the [Backport
-Wishlist](./Backport-Wishlist.md).
+This page describes the policy GlusterFS has regarding the backports. As a user, or contributor, being aware of this policy would help you to understand how to request for backport from community.
```

Comment:
It's better to stick to 80 character limit wherever possible - here and elsewhere in the PR. The hyperlinks are an exception though. Markdown doesn't interpret single line break as a new line during rendering.

Reply:
Done